.github/workflows/ci.yml

```diff
@@ -10,6 +10,25 @@ env:
   CARGO_TERM_COLOR: always
 
 jobs:
+  fmt:
+    runs-on: "ubuntu-latest"
+    steps:
+      - uses: Swatinem/rust-cache@v2
+      - uses: actions/checkout@v3
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: 1.69.0
+          components: rustfmt, clippy
+      - name: Setup WasmEdge build env
+        run: |
```

Comment:
Looks like we need all the system libs like in line 48 here.
Probably not build-essential but likely everything else.

Reply:
Done.